src/sentry/static/sentry/app/views/organizationDiscover/queryBuilder.jsx

```diff
@@ -193,6 +195,38 @@ export default function createQueryBuilder(initial = {}, organization) {
       });
   }
 
+  /**
+   * Get the actual query to be run for each visualization type
+   *
+   * @param {Object} originalQuery Original query input by user (external query representation)
+   * @param {String} Type to fetch - currently either byDay or base
+   * @returns {Object} Modified query to be run for that type
+   */
+  function getQueryByType(originalQuery, type) {
+    if (type === 'byDayQuery') {
```

Comment:
Do we want these to be defined as constants somewhere?

Reply:
I'll leave it for now, but might be good to follow up later when we end up with a number of these